samza-core/src/main/java/org/apache/samza/processor/StreamProcessor.java

```diff
@@ -206,16 +219,18 @@ public void onJobModelExpired() {
           if (SamzaContainerStatus.NOT_STARTED.equals(status) || SamzaContainerStatus.STARTED.equals(status)) {
             boolean shutdownComplete = false;
             try {
-              LOGGER.info("Shutting down container in onJobModelExpired.");
+              LOGGER.info("Shutting down container in onJobModelExpired for processor:" + pid);
               container.pause();
               shutdownComplete = jcContainerShutdownLatch.await(taskShutdownMs, TimeUnit.MILLISECONDS);
+              LOGGER.info("Latch " + jcContainerShutdownLatch + " returned. ShutdownCompete=" + shutdownComplete + ";pid=" + pid + ";cd=" + jcContainerShutdownLatch.getCount());
```

Comment:
I will delete it. I was using it for debugging and needed hash value to make sure it was the same object.